subxt/src/tx/tx_client.rs

```diff
@@ -103,11 +103,14 @@ impl<T: Config, C: OfflineClientT<T>> TxClient<T, C> {
     }
 
     /// Create a partial extrinsic.
-    pub fn create_partial_signed_with_nonce<Call>(
+    ///
+    /// Note: This function will **not** refine the params to use the latest account nonce and the latest block hash/number.
+    /// Instead your call will get default values of nonce = 0 and mortality = Immortal for the signed extensions, if the params
+    /// have not been 'refined' before.
```

Comment:
```suggestion
    /// Note: if not provided, the default account nonce will be set to 0 and the default mortality will be _immortal_.
    /// This is because this method runs offline, and so is unable to fetch the data needed for more appropriate values.
```

Just to make it clear that because offline, we have to use worse defaults, and to avoid mentioning any new concept like "refining params"